db/migrate/20220801044610_add_outbound_message_id_to_post.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+class AddOutboundMessageIdToPost < ActiveRecord::Migration[7.0]
+  def change
+    if !column_exists?(:posts, :outbound_message_id)
```

Comment:
What is the reason we need this conditional here? The column has not been added and dropped before so it doesn't seem like this guard will buy us anything.

Reply:
I think I did this when I was thinking about doing experimental rollout deploys of this, will remove.